packages/components/inputs/toggle-input/src/toggle-input.styles.ts

```diff
@@ -125,10 +125,19 @@ const getNewThemeFocusIndicator = (props: TStyledSpanProps) => css`
   }
 `;
 
+const getNewThemeMargin = (props: TStyledLabelProps) =>
+  css`
+    margin: ${props.isNewTheme
+      ? `${props.size === 'big' ? '14px' : '9px'} !important`
```

Comment:
What's the rationale behind these "magic" numbers? How are they being computed?

Reply:
Oh yes, thank you, it's definitely worth explaining. Updated in 4a0a22bcb667bb9521ede104a9a6fbbf72146fb4